.docs/content/1.installation/1.linux/2.all-in-one-deployment.md

```diff
@@ -0,0 +1,49 @@
+# All in one deployment
```

Comment:
How does this is different from the file 1.deployment? I think this part is missing because if I follow docs, I will deploy using script in 1.deployment and then coming here and read about another deployment?

Reply:
from what I understand, it doesn't just launch the localhost deployment but also all the cloud services. Maybe you are suggesting that we name which services?

Reply:
Yes, we can do that.